README.md

```diff
@@ -194,6 +194,16 @@ of more than one.
 
 `bundle exec rake`
 
+### Testing in a browser
+
+If you want to test this behaviour in a browser without Fastly set up (in dev
```

Comment:
I completely forgot it did that (and I've been running on localhost as I can't get government-frontend to work at the moment on dev).

In that case this PR should change to mention using the GOV.UK Toolkit extension instead.